src/mock-server.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express')
```

Comment:
This was put in simply for mocking the API calls, I would expect it to be removed once we have a proper API in place.

Reply:
It is possibly overkill for mock purposes, depends on future use. It does keep some separation between the temporary/production code which is nice. Could be worth adding in a separate PR because of that?